web-console/src/utils/ingestion-spec.tsx

```diff
@@ -2663,7 +2663,7 @@ function guessInputFormat(sampleData: string[]): InputFormat {
       return inputFormatFromType('orc');
     }
     // Avro OCF 4 byte magic header: https://avro.apache.org/docs/current/spec.html#Object+Container+Files
-    if (sampleDatum.startsWith('Obj1')) {
+    if (sampleDatum.startsWith('Obj')) {
```

Comment:
It'd be really great if we could test the proper thing here, but I'm not sure how either.

@vogievetsky Any ideas about how we could actually check all 4 bytes here? (See https://github.com/apache/druid/issues/10229; the "1" should actually be the unprintable character 0x01.)

Also, are there tests for this logic block in general? Where could we add one for this particular case?

Reply:
sure thing. just do `if (sampleDatum.startsWith('Obj') && sampleDatum.charCodeAt(3) === 1)` 

![image](https://user-images.githubusercontent.com/177816/90039386-6746c200-dc7b-11ea-88cc-f0755fb79287.png)